lte/gateway/python/setup.py

```diff
@@ -125,7 +125,7 @@ def os_release():
             'idna==2.8',
             'python-dateutil==2.8.1',
             'six>=1.12.0',
-            'eventlet>=0.24',
+            'eventlet==0.30.2',
```

Comment:
do we want `==` or `>=`?

Reply:
pipelineD is having issue with 0.31. so the dependency is fixed at 0.30.2.